melos.yaml

```diff
@@ -12,35 +12,181 @@ ignore:
 scripts:
   setup_tuneup: >
     flutter pub global activate tuneup
-
   copy_dummy_config: >
     melos exec --scope="*example*,sample_app" -- \
       cp -n "\$MELOS_ROOT_PATH"/.circleci/dummy_amplifyconfiguration.dart lib/amplifyconfiguration.dart | true
+  build:examples:release:ios: >
+    melos exec -c 1 --scope="*example*" --ignore="*_android*" --fail-fast -- \
+      flutter build ios --no-codesign
+
+  build:examples:release:android: >
+    melos exec -c 1 --scope="*example*" --ignore="*_ios*" --fail-fast -- \
+      flutter build apk --verbose
+
+  test:unit:flutter:
+    run: ./.circleci/test_all_plugins.sh flutter-test $1 $PWD
+
+  test:unit:android:
```

Comment:
For the platform ones, might need to list the packages in scope (like analyze command) or otherwise use scope command. Still, would hope to call this command `melos run:test:unit:android` and then it prompts me to select the package, or run them all. CI would obvi have to execute in a single plugin.